pallets/moonbeam-foreign-assets/src/lib.rs

```diff
@@ -274,18 +322,25 @@ pub mod pallet {
 			let name = core::str::from_utf8(&name).map_err(|_| Error::<T>::InvalidTokenName)?;
 
 			let contract_address = EvmCaller::<T>::erc20_create(asset_id, decimals, symbol, name)?;
+			let owner = AssetOwner::<T>::Governance;
 
 			// Insert the association assetId->foreigAsset
 			// Insert the association foreigAsset->assetId
 			AssetsById::<T>::insert(&asset_id, &xcm_location);
 			AssetsByLocation::<T>::insert(&xcm_location, (asset_id, AssetStatus::Active));
-
-			T::OnForeignAssetCreated::on_asset_created(&xcm_location, &asset_id);
+			AssetsCreationDetails::<T>::insert(
+				&asset_id,
+				AssetCreationDetails {
+					owner,
+					deposit: None,
+				},
+			);
```

Comment:
How important is this storage for assets that were created with governance? Asking because the assets were already migrated on stagenet, and I do not see it being breaking if we don't `fix` the storage there.

Seems only important for assets created by normal accounts.